client/signup/steps/design-type-with-store/siteground-store/siteground-logo.jsx

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
My guess is that this should probably be added to `SocialLogo` or something like that - you might check with someone on that.